bundles/EcommerceFrameworkBundle/PaymentManager/V7/Payment/StartPaymentResponse/HobexResponse.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\EcommerceFrameworkBundle\PaymentManager\V7\Payment\StartPaymentResponse;
+
+class HobexResponse implements StartPaymentResponseInterface
```

Comment:
why not use [SnippetResponse](https://github.com/pimcore/pimcore/blob/68046984953e4a69dc6ff5cdf3cfb7a2e7e8af77/bundles/EcommerceFrameworkBundle/PaymentManager/V7/Payment/StartPaymentResponse/SnippetResponse.php#L41) here?

Reply:
at least as parent class, then rendering the form would be more unified.

Reply:
You are right. I replaced the HobexResponse with SnippetResponse: https://github.com/pimcore/pimcore/pull/6804/commits/5653cb38da485bd8f7f66485600c2fa1f9f242d3